Gems/ScriptCanvas/Code/Editor/SystemComponent.cpp

```diff
@@ -130,6 +130,7 @@ namespace ScriptCanvasEditor
         ScriptCanvasExecutionBus::Handler::BusConnect();
         AzToolsFramework::EditorEvents::Bus::Handler::BusConnect();
         AzToolsFramework::AssetBrowser::AssetBrowserInteractionNotificationBus::Handler::BusConnect();
+        AzToolsFramework::EditorEntityContextNotificationBus::Handler::BusConnect();
```

Comment:
question: what is this for?

Reply:
It connects to events Start/Stop play in editor, which is when the system component requests a full Lua garbage collet.